isaric/parsers/uganda-vri.toml

```diff
@@ -0,0 +1,476 @@
+  context = ['walk on a flat surface for 20 mins']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "date_30" }
+  phase = "followup"
+  text = { field = "stairs_v2", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk up a hill']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "date_60" }
+  phase = "followup"
+  text = { field = "move_v60", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk around the home']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "date_60" }
+  phase = "followup"
+  text = { field = "walk_v60", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk on a flat surface for 20 mins']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "date_60" }
+  phase = "followup"
+  text = { field = "stairs_v60", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk up a hill']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  text = { field = "assist_study", ref = "Y/N" }
+  context = ['requires assistance to walk/get out of bed']
+
+[[observation]]
+  name = "lower_chest_wall_indrawing"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "indraw", ref = "Y/N" }
+
+[[observation]]
+  name = "runny_nose"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "runny_nose", ref = "Y/N" }
+
+[[observation]]
+  name = "seizures"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "seizure", ref = "Y/N" }
+
+[[observation]]
+  name = "shortness_of_breath"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "breath", ref = "Y/N" }
+
+[[observation]]
+  name = "skin_rash"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "rash_skin", ref = "Y/N" }
+
+[[observation]]
+  name = "sore_throat"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "throat", ref = "Y/N" }
+
+[[observation]] # SHOULD WE MAKE THIS A MIN OR MAX OF the recorded values? i.e., minimum(caps_1, caps_2, caps_3)
+  name = "sternal_capillary_refill_time_greater_2s"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "caps_1" }
+
+[[observation]]
+  name = "sternal_capillary_refill_time_greater_2s"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "caps_2" }
+
+[[observation]]
+  name = "sternal_capillary_refill_time_greater_2s"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "caps_3" }
```

Comment:
I would make it a 'max' - a longer refill time indicates worse circulation - and then add a context note stating that.